### PR TITLE
Add com.snowplowanalytics.dataflowrunner/ClusterConfig/avro/1-1-0

### PR DESCRIPTION
**Pull request by BenFradet**
- [x] applications
- [x] EBS

### CODE REVIEW

File: schemas/com.snowplowanalytics.dataflowrunner/ClusterConfig/avro/1-1-0 [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+                  "type": {
+                    "name": "MasterRecord",
+                    "type": "record",
+                    "fields": [
+                      {
+                        "name": "type",
+                        "type": "string"
+                      },
+                      {
+                        "name": "ebsConfiguration",
```
Comment (BenFradet) [inline schemas/com.snowplowanalytics.dataflowrunner/ClusterConfig/avro/1-1-0]: added this

File: schemas/com.snowplowanalytics.dataflowrunner/ClusterConfig/avro/1-1-0 [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+        }
+      }, "null"]
+    },
+    {
+      "name": "applications",
+      "type": [{
+        "type": "array",
+        "items": "string"
+      }, "null"]
+    }
```
Comment (BenFradet) [inline schemas/com.snowplowanalytics.dataflowrunner/ClusterConfig/avro/1-1-0]: and this

**Comment by alexanderdean** [comment]
LGTM but please clean up the git history!

**Comment by BenFradet** [comment]
done :+1:

**Comment by alexanderdean** [comment]
Nice one, scheduled

**Comment by alexanderdean** [comment]
Will auto-close on merge to master

**Pull request closed without merging**